### PR TITLE
Add com.github.danrabbit.harvey

### PR DESCRIPTION
**Pull request by alex285**
Harvey repo
https://github.com/danrabbit/harvey

### CODE REVIEW

File: com.github.danrabbit.harvey/com.github.danrabbit.harvey.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+id: com.github.danrabbit.harvey
+runtime: org.gnome.Platform
+runtime-version: '3.28'
+sdk: org.gnome.Sdk
+command: com.github.danrabbit.harvey
+rename-icon: com.github.danrabbit.harvey
```
Comment (jurf) [inline com.github.danrabbit.harvey/com.github.danrabbit.harvey.yml]: `com.github.danrabbit.harvey` is already the ID, so this won’t have any effect.

File: com.github.danrabbit.harvey/com.github.danrabbit.harvey.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+tags: ['contrast', 'color']
```
Comment (jurf) [inline com.github.danrabbit.harvey/com.github.danrabbit.harvey.yml]: I would leave these out, this is AppData material, not standard in other apps as well.

File: com.github.danrabbit.harvey/com.github.danrabbit.harvey.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+finish-args:
+  - '--share=ipc'
+  - '--socket=x11'
+  - '--socket=wayland'
+  - '--talk-name=ca.desrt.dconf'
+  - '--env=DCONF_USER_CONFIG_DIR=.config/dconf'
+  - '--filesystem=xdg-run/dconf'
+  - '--filesystem=~/.config/dconf:ro'
+  - '--own-name=com.github.danrabbit.harvey'
```
Comment (jurf) [inline com.github.danrabbit.harvey/com.github.danrabbit.harvey.yml]: Again, the ID seems to be correct, this shouldn’t be needed. Try removing it and seeing if everything still runs OK.

File: com.github.danrabbit.harvey/com.github.danrabbit.harvey.yml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+cleanup:
+  - '*.a'
+  - '*.h'
```
Comment (jurf) [inline com.github.danrabbit.harvey/com.github.danrabbit.harvey.yml]: I’m pretty sure these only end up in `/include`, so this mightn’t be needed, but I’ll check the build output. Also, if `.a` files get installed, you might want to check if there isn’t something along the lines of `--disable-static` in the build options of the app.

**Comment by jurf** [comment]
bot, build

**Comment by jurf** [comment]
Also, maybe something like this will be needed: flathub/com.github.robertsanseries.ciano#2.

**Comment by jurf** [comment]
bot, build com.github.danrabbit.harvey

**Comment by alex285** [comment]
sorry for the single pushes, i edit directly on Github

**Comment by jurf** [comment]
Oh, wait – figured out what [this](https://flathub.org/builds/#/builders/2/builds/4692) error means. You’re supposed to make a PR against the `new-pr` branch, not `master`. :slightly_smiling_face: I’m not sure  GitHub allows changing the branch like this, but I doubt it, you’ll probably need to make a new PR.

**Comment by alex285** [comment]
> Oh, wait – figured out what [this](https://flathub.org/builds/#/builders/2/builds/4692) error means. You’re supposed to make a PR against the `new-pr` branch, not `master`. slightly_smiling_face I’m not sure GitHub allows changing the branch like this, but I doubt it, you’ll probably need to make a new PR.

noticed. i think i should close that PR and make a new one?

**Comment by jurf** [comment]
[Yes](https://stackoverflow.com/questions/24159036/how-to-modify-a-pull-request-on-github-to-change-target-branch-to-merge-into#24159161) and yes. The YAML file gets converted to JSON internally anyway.

**Comment by alex285** [comment]
@jurf sorry about this
new PR: https://github.com/flathub/flathub/pull/639

**Comment by jurf** [comment]
No problem dude. :slightly_smiling_face:

**Pull request closed without merging**